### ISSUE TITLE
`awaited` should be removed from keywords

### ISSUE THREAD

**Issue by Zzzen** [microsoft/TypeScript-TmLanguage#874]
**[TS and JS Grammar Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.typescript-javascript-grammar) version:**  0.0.55 / latest

**Code**

```ts
// Please include a code snippet that demonstrates the issue
type N = awaited Promise<number>;
type NN = randomText Promise<number>;
```

![image](https://user-images.githubusercontent.com/6630042/123895321-37d39d80-d992-11eb-9569-6d9d90e47307.png)

`awaited` is highlighted incorrectly in vscode, which makes me think it's a keyword.

**Comment by orta** [comment]
It is a keyword in TypeScript https://github.com/microsoft/TypeScript/pull/35998 👍🏻

**Comment by Zzzen** [comment]
😏   That's how I thought yesterday and got stuck by `Cannot find name 'awaited'`. But actually, it has been reverted. There no `SyntaxKind.AwaitedKeyword` anymore.

**Comment by orta** [comment]
Strange, there's still tests in the compiler for it...

**Comment by orta** [comment]
Thanks for the PR @Zzzen  - fixed in #875
